src/System.Private.CoreLib/src/System/MissingFieldException.CoreCLR.cs

```diff
@@ -23,15 +19,8 @@ public override string Message
         {
             get
             {
-                if (ClassName == null)
-                {
-                    return base.Message;
-                }
-                else
-                {
-                    // do any desired fixups to classname here.
-                    return SR.Format(SR.MissingField_Name, (Signature != null ? FormatSignature(Signature) + " " : "") + ClassName + "." + MemberName);
-                }
+                // do any desired fixups to classname here.
+                return ClassName == null ? base.Message : SR.Format(SR.MissingField_Name, (Signature != null ? FormatSignature(Signature) + " " : "") + ClassName + "." + MemberName);
```

Comment:
This is what `Message` getter looks like in corert:

```
return ClassName == null ? base.Message : SR.Format(SR.MissingField_Name, ClassName + "." + MemberName + (Signature != null ? " " + FormatSignature(Signature) : string.Empty));
```

Reply:
This is the only reason I have this class as partial

Reply:
I think that this is just a copy&paste mistake in CoreRT. It should be fine to use the CoreCLR version for the shared one.